PrBuild/config.json

```diff
@@ -1,12 +1,14 @@
-{      
-    "repo_owner": "pniko",
-    "repo_name": "detox-appcenter",
-
-    "appcenter_owner_type": "users",
-    "appcenter_owner": "pniko",
-    "appcenter_app": "detox-appcenter",
-
-    "branch_template": "master",
-    "branch_regex": "(skip|no-build)"
+{
+    "repos": [
+        {
+            "repo_owner": "pniko",
+            "repo_name": "detox-appcenter",
+            "branch_template": "master",
+            "branch_regex": "(skip|no-build)",
```

Comment:
Let's drop this for now. I was planning to put in a regex to skip rebuilding on commits that match this pattern, but it doesn't come in the PR payload for the webhook from Github. We'll decompose this into some other set of configuration options.

Reply:
Got it, fixed, thanks for the clarification.